---a/net/minecraft/item/ItemSnow.java
+++b/net/minecraft/item/ItemSnow.java
@@ -8,49 +8,39 @@
 import net.minecraft.util.EnumFacing;
 import net.minecraft.world.World;
 
-public class ItemSnow extends ItemBlock
-{
+public class ItemSnow extends ItemBlock {
+
     private static final String __OBFID = "CL_00000068";
 
-    public ItemSnow(Block block)
-    {
+    public ItemSnow(Block block) {
         super(block);
         this.setMaxDamage(0);
         this.setHasSubtypes(true);
     }
 
-    public boolean onItemUse(ItemStack stack, EntityPlayer playerIn, World worldIn, BlockPos pos, EnumFacing side, float hitX, float hitY, float hitZ)
-    {
-        if (stack.stackSize == 0)
-        {
+    public boolean onItemUse(ItemStack stack, EntityPlayer playerIn, World worldIn, BlockPos pos, EnumFacing side, float hitX, float hitY, float hitZ) {
+        if (stack.stackSize == 0) {
             return false;
-        }
-        else if (!playerIn.canPlayerEdit(pos, side, stack))
-        {
+        } else if (!playerIn.canPlayerEdit(pos, side, stack)) {
             return false;
-        }
-        else
-        {
+        } else {
             IBlockState iblockstate = worldIn.getBlockState(pos);
             Block block = iblockstate.getBlock();
 
-            if (block != this.block && side != EnumFacing.UP)
-            {
+            if (block != this.block && side != EnumFacing.UP) {
                 pos = pos.offset(side);
                 iblockstate = worldIn.getBlockState(pos);
                 block = iblockstate.getBlock();
             }
 
-            if (block == this.block)
-            {
-                int i = ((Integer)iblockstate.getValue(BlockSnow.LAYERS)).intValue();
+            if (block == this.block) {
+                int i = ((Integer) iblockstate.getValue(BlockSnow.LAYERS)).intValue();
 
-                if (i <= 7)
-                {
+                if (i <= 7) {
                     IBlockState iblockstate1 = iblockstate.withProperty(BlockSnow.LAYERS, Integer.valueOf(i + 1));
 
-                    if (worldIn.checkNoEntityCollision(this.block.getCollisionBoundingBox(worldIn, pos, iblockstate1)) && worldIn.setBlockState(pos, iblockstate1, 2))
-                    {
-                        worldIn.playSoundEffect((double)((float)pos.getX() + 0.5F), (double)((float)pos.getY() + 0.5F), (double)((float)pos.getZ() + 0.5F), this.block.stepSound.getPlaceSound(), (this.block.stepSound.getVolume() + 1.0F) / 2.0F, this.block.stepSound.getFrequency() * 0.8F);
+                    if (worldIn.checkNoEntityCollision(this.block.getCollisionBoundingBox(worldIn, pos, iblockstate1)) && worldIn.setBlockState(pos, iblockstate1, 2)) {
+                        worldIn.playSoundEffect((double) ((float) pos.getX() + 0.5F), (double) ((float) pos.getY() + 0.5F), (double) ((float) pos.getZ() + 0.5F), this.block.stepSound.getPlaceSound(), (this.block.stepSound.getVolume() + 1.0F) / 2.0F, this.block.stepSound.getFrequency() * 0.8F);
                         --stack.stackSize;
                         return true;
                     }
@@ -62,9 +62,7 @@
         }
     }
 
-    public int getMetadata(int damage)
-    {
+    public int getMetadata(int damage) {
         return damage;
     }
-}
+}
